pwa/src/components/deleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import "./deletemodal.css";
+import { Modal } from "@conductionnl/nl-design-system";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+interface DeleteModalProps {
+  handleDelete: any;
+  handleId: any;
```

Comment:
I would recommend renaming this.

1. I'm unsure if you mean a _handler_ here (as in the resource coupled to endpoints), if you do, this would be way too specific because this delete modal has nothing to do with the handlers or
2. If you meant it as something to _handle_ (like _handleDelete_ handles the deletion of something): this is not actually handling anything (it's not a function).

Therefore, I would rename it to something like `resourceId`.

Reply:
Edit: and it should't be `:any`.